kubetest/main.go

```diff
@@ -56,69 +56,72 @@ var (
-	federation          bool
-	flushMemAfterBuild  bool
-	focusRegex          string
-	gcpCloudSdk         string
-	gcpMasterImage      string
-	gcpMasterSize       string
-	gcpNetwork          string
-	gcpNodeImage        string
-	gcpImageFamily      string
-	gcpImageProject     string
-	gcpNodes            string
-	gcpNodeSize         string
-	gcpProject          string
-	gcpProjectType      string
-	gcpServiceAccount   string
-	gcpRegion           string
-	gcpZone             string
-	ginkgoParallel      ginkgoParallelValue
-	kubecfg             string
-	kubemark            bool
-	kubemarkMasterSize  string
-	kubemarkNodes       string // TODO(fejta): switch to int after migration
-	logexporterGCSPath  string
-	metadataSources     string
-	multiClusters       multiClusterDeployment
-	multipleFederations bool
-	noAllowDup          bool
-	nodeArgs            string
-	nodeTestArgs        string
-	nodeTests           bool
-	perfTests           bool
-	provider            string
-	publish             string
-	runtimeConfig       string
-	save                string
-	skew                bool
-	skipRegex           string
-	soak                bool
-	soakDuration        time.Duration
-	sshUser             string
-	stage               stageStrategy
-	stageFederation     stageFederationStrategy
-	test                bool
-	testArgs            string
-	testCmd             string
-	testCmdName         string
-	testCmdArgs         []string
-	up                  bool
-	upgradeArgs         string
+	build              	 buildStrategy
+	buildFederation    	 buildFederationStrategy
+	charts             	 bool
+	checkLeaks         	 bool
+	checkSkew          	 bool
+	cluster            	 string
+	clusterIPRange     	 string
+	deployment          	 string
+	dindImage          	 string
+	down              	 bool
+	dump               	 string
+	extract            	 extractStrategies
+	extractFederation  	 extractFederationStrategies
+	extractSource      	 bool
+	federation         	 bool
+	flushMemAfterBuild 	 bool
+	focusRegex         	 string
+	gcpCloudSdk        	 string
+	gcpMasterImage     	 string
+	gcpMasterSize      	 string
+	gcpNetwork         	 string
+	gcpNodeImage       	 string
+	gcpImageFamily     	 string
+	gcpImageProject    	 string
+	gcpNodes           	 string
+	gcpNodeSize        	 string
+	gcpProject         	 string
+	gcpProjectType     	 string
+	gcpServiceAccount  	 string
+	gcpRegion          	 string
+	gcpZone            	 string
+	ginkgoParallel     	 ginkgoParallelValue
+	kubecfg            	 string
+	kubemark           	 bool
+	kubemarkMasterSize 	 string
+	kubemarkNodes      	 string // TODO(fejta): switch to int after migration
+	logexporterGCSPath 	 string
+	metadataSources    	 string
+	multiClusters      	 multiClusterDeployment
+	multipleFederations	 bool
+	noAllowDup         	 bool
+	nodeArgs           	 string
+	nodeTestArgs       	 string
+	nodeTests          	 bool
+        perfTestsClusterLoader   bool
```

Comment:
Included flags to call clusterloader,netperf and dns.
Other lines are modified to have proper alignment